auth/src/user.rs

```diff
@@ -376,16 +456,25 @@ where
     }
 }
 
-impl From<User> for shuttle_common::models::user::Response {
+impl From<User> for models::user::Response {
     fn from(user: User) -> Self {
         Self {
             name: user.name.to_string(),
             key: user.key.expose().as_ref().to_owned(),
             account_tier: user.account_tier.to_string(),
-            // TODO: this just returns what was always returned, the id of the pro subscription.
-            // We will need to update this when we want to also return the rds subscription. We
-            // can return a vec of IDs, but it will be a breaking change for the console (I don't believe it is used anywhere else).
-            subscription_id: user.pro_subscription_id().map(|id| id.to_string()),
+            subscriptions: user.subscriptions.into_iter().map(Into::into).collect(),
```

Comment:
This ended up being a breaking change for the console, right? We need to ensure that it's updated over there as well, but you probably talked with Guga about this already.

Reply:
Yip, the console update is in the works